src/lib/features/project/project-owners-read-model.ts

```diff
@@ -44,7 +28,7 @@ export class ProjectOwnersReadModel {
     ): IProjectWithCountAndOwners[] {
         return projects.map((project) => ({
             ...project,
-            owners: owners[project.name] || [{ ownerType: 'system' }],
+            owners: owners[project.id] || [{ ownerType: 'system' }],
```

Comment:
Good catch here. Did we really use project name? 😬

Reply:
yup. it took me couple of minutes to debug xD